deployment/A_introduction.md

```diff
@@ -59,12 +59,12 @@ $ MIX_ENV=prod mix compile
 
 ## Compiling your application assets
 
-This step is required only if you have static assets like images, javascript, stylesheets and more in your Phoenix applications. By default, Phoenix uses brunch, and that's what we are going to explore.
+This step is required only if you have static assets like images, javascript, stylesheets and more in your Phoenix applications. By default, Phoenix uses brunch, and that's what we are going to explore. You don't need a local installation of brunch, just run ```npm run deploy``` and it will invoke ```brunch build --production``` for you.
```

Comment:
I think the "You don't need a local installation of brunch" is a little misleading, it still needs to be installed.